Description: account for the "github.com/go-check/check" -> "gopkg.in/check.v1" rename
Author: Tianon Gravi <tianon@debian.org>
Forwarded: not-needed
Notes: find -name '*.go' -exec sed -i 's!github.com/go-check/check!gopkg.in/check.v1!g' '{}' + && git checkout -- vendor && git diff -- '*.go' > check-v1.patch

diff --git a/integration-cli/benchmark_test.go b/integration-cli/benchmark_test.go
index 647d014..3398305 100644
--- a/integration-cli/benchmark_test.go
+++ b/integration-cli/benchmark_test.go
@@ -9,7 +9,7 @@ import (
 	"sync"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) BenchmarkConcurrentContainerActions(c *check.C) {
diff --git a/integration-cli/check_test.go b/integration-cli/check_test.go
index aab8e4a..971ac31 100644
--- a/integration-cli/check_test.go
+++ b/integration-cli/check_test.go
@@ -8,7 +8,7 @@ import (
 
 	"github.com/docker/docker/cliconfig"
 	"github.com/docker/docker/pkg/reexec"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func Test(t *testing.T) {
diff --git a/integration-cli/daemon.go b/integration-cli/daemon.go
index 2e3985a..f6f5f55 100644
--- a/integration-cli/daemon.go
+++ b/integration-cli/daemon.go
@@ -18,7 +18,7 @@ import (
 	"github.com/docker/docker/pkg/ioutils"
 	"github.com/docker/docker/pkg/tlsconfig"
 	"github.com/docker/go-connections/sockets"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Daemon represents a Docker daemon for the testing framework.
diff --git a/integration-cli/docker_api_attach_test.go b/integration-cli/docker_api_attach_test.go
index a0a8e7d..620f5a9 100644
--- a/integration-cli/docker_api_attach_test.go
+++ b/integration-cli/docker_api_attach_test.go
@@ -9,7 +9,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"golang.org/x/net/websocket"
 )
 
diff --git a/integration-cli/docker_api_auth_test.go b/integration-cli/docker_api_auth_test.go
index 63e78ab..6445064 100644
--- a/integration-cli/docker_api_auth_test.go
+++ b/integration-cli/docker_api_auth_test.go
@@ -5,7 +5,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/engine-api/types"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Test case for #22244
diff --git a/integration-cli/docker_api_build_test.go b/integration-cli/docker_api_build_test.go
index 49de71c..7132900 100644
--- a/integration-cli/docker_api_build_test.go
+++ b/integration-cli/docker_api_build_test.go
@@ -6,7 +6,7 @@ import (
 	"net/http"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestBuildApiDockerfilePath(c *check.C) {
diff --git a/integration-cli/docker_api_containers_test.go b/integration-cli/docker_api_containers_test.go
index 2622a54..2249ed1 100644
--- a/integration-cli/docker_api_containers_test.go
+++ b/integration-cli/docker_api_containers_test.go
@@ -21,7 +21,7 @@ import (
 	"github.com/docker/engine-api/types"
 	containertypes "github.com/docker/engine-api/types/container"
 	networktypes "github.com/docker/engine-api/types/network"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestContainerApiGetAll(c *check.C) {
diff --git a/integration-cli/docker_api_create_test.go b/integration-cli/docker_api_create_test.go
index d29b355..727d971 100644
--- a/integration-cli/docker_api_create_test.go
+++ b/integration-cli/docker_api_create_test.go
@@ -5,7 +5,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestApiCreateWithNotExistImage(c *check.C) {
diff --git a/integration-cli/docker_api_events_test.go b/integration-cli/docker_api_events_test.go
index cb219fb..b5a6a30 100644
--- a/integration-cli/docker_api_events_test.go
+++ b/integration-cli/docker_api_events_test.go
@@ -11,7 +11,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/jsonmessage"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestEventsApiEmptyOutput(c *check.C) {
diff --git a/integration-cli/docker_api_exec_resize_test.go b/integration-cli/docker_api_exec_resize_test.go
index 2c0c876..cf56a42 100644
--- a/integration-cli/docker_api_exec_resize_test.go
+++ b/integration-cli/docker_api_exec_resize_test.go
@@ -10,7 +10,7 @@ import (
 	"sync"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestExecResizeApiHeightWidthNoInt(c *check.C) {
diff --git a/integration-cli/docker_api_exec_test.go b/integration-cli/docker_api_exec_test.go
index f16582f..c8729ab 100644
--- a/integration-cli/docker_api_exec_test.go
+++ b/integration-cli/docker_api_exec_test.go
@@ -11,7 +11,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Regression test for #9414
diff --git a/integration-cli/docker_api_images_test.go b/integration-cli/docker_api_images_test.go
index 9d35b0c..0bee32c 100644
--- a/integration-cli/docker_api_images_test.go
+++ b/integration-cli/docker_api_images_test.go
@@ -8,7 +8,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/engine-api/types"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestApiImagesFilter(c *check.C) {
diff --git a/integration-cli/docker_api_info_test.go b/integration-cli/docker_api_info_test.go
index 9e6af66..a154c15 100644
--- a/integration-cli/docker_api_info_test.go
+++ b/integration-cli/docker_api_info_test.go
@@ -4,7 +4,7 @@ import (
 	"net/http"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestInfoApi(c *check.C) {
diff --git a/integration-cli/docker_api_inspect_test.go b/integration-cli/docker_api_inspect_test.go
index 6b55159..e4a2421 100644
--- a/integration-cli/docker_api_inspect_test.go
+++ b/integration-cli/docker_api_inspect_test.go
@@ -9,7 +9,7 @@ import (
 	"github.com/docker/docker/pkg/stringutils"
 	"github.com/docker/engine-api/types"
 	"github.com/docker/engine-api/types/versions/v1p20"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestInspectApiContainerResponse(c *check.C) {
diff --git a/integration-cli/docker_api_inspect_unix_test.go b/integration-cli/docker_api_inspect_unix_test.go
index fe59860..5a9c8a3 100644
--- a/integration-cli/docker_api_inspect_unix_test.go
+++ b/integration-cli/docker_api_inspect_unix_test.go
@@ -8,7 +8,7 @@ import (
 	"net/http"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // #16665
diff --git a/integration-cli/docker_api_logs_test.go b/integration-cli/docker_api_logs_test.go
index 2ff27f8..c71b1cf 100644
--- a/integration-cli/docker_api_logs_test.go
+++ b/integration-cli/docker_api_logs_test.go
@@ -9,7 +9,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestLogsApiWithStdout(c *check.C) {
diff --git a/integration-cli/docker_api_network_test.go b/integration-cli/docker_api_network_test.go
index e65c7b5..967a7d2 100644
--- a/integration-cli/docker_api_network_test.go
+++ b/integration-cli/docker_api_network_test.go
@@ -12,7 +12,7 @@ import (
 	"github.com/docker/engine-api/types"
 	"github.com/docker/engine-api/types/filters"
 	"github.com/docker/engine-api/types/network"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestApiNetworkGetDefaults(c *check.C) {
diff --git a/integration-cli/docker_api_resize_test.go b/integration-cli/docker_api_resize_test.go
index 73023dd..9466e71 100644
--- a/integration-cli/docker_api_resize_test.go
+++ b/integration-cli/docker_api_resize_test.go
@@ -5,7 +5,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestResizeApiResponse(c *check.C) {
diff --git a/integration-cli/docker_api_stats_test.go b/integration-cli/docker_api_stats_test.go
index 7c3f8d3..40082d6 100644
--- a/integration-cli/docker_api_stats_test.go
+++ b/integration-cli/docker_api_stats_test.go
@@ -13,7 +13,7 @@ import (
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/version"
 	"github.com/docker/engine-api/types"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 var expectedNetworkInterfaceStats = strings.Split("rx_bytes rx_dropped rx_errors rx_packets tx_bytes tx_dropped tx_errors tx_packets", " ")
diff --git a/integration-cli/docker_api_test.go b/integration-cli/docker_api_test.go
index a725777..cb22bb8 100644
--- a/integration-cli/docker_api_test.go
+++ b/integration-cli/docker_api_test.go
@@ -12,7 +12,7 @@ import (
 
 	"github.com/docker/docker/api"
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestApiOptionsRoute(c *check.C) {
diff --git a/integration-cli/docker_api_update_unix_test.go b/integration-cli/docker_api_update_unix_test.go
index 607e76a..a5b69ba 100644
--- a/integration-cli/docker_api_update_unix_test.go
+++ b/integration-cli/docker_api_update_unix_test.go
@@ -6,7 +6,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestApiUpdateContainer(c *check.C) {
diff --git a/integration-cli/docker_api_version_test.go b/integration-cli/docker_api_version_test.go
index ccb1484..d71a6bb 100644
--- a/integration-cli/docker_api_version_test.go
+++ b/integration-cli/docker_api_version_test.go
@@ -7,7 +7,7 @@ import (
 	"github.com/docker/docker/dockerversion"
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/engine-api/types"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestGetVersion(c *check.C) {
diff --git a/integration-cli/docker_api_volumes_test.go b/integration-cli/docker_api_volumes_test.go
index 732271d..d4ed266 100644
--- a/integration-cli/docker_api_volumes_test.go
+++ b/integration-cli/docker_api_volumes_test.go
@@ -7,7 +7,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/engine-api/types"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestVolumesApiList(c *check.C) {
diff --git a/integration-cli/docker_cli_attach_test.go b/integration-cli/docker_cli_attach_test.go
index 0ac3e1a..91934ad 100644
--- a/integration-cli/docker_cli_attach_test.go
+++ b/integration-cli/docker_cli_attach_test.go
@@ -10,7 +10,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 const attachWait = 5 * time.Second
diff --git a/integration-cli/docker_cli_attach_unix_test.go b/integration-cli/docker_cli_attach_unix_test.go
index 7af761d..b5eab25 100644
--- a/integration-cli/docker_cli_attach_unix_test.go
+++ b/integration-cli/docker_cli_attach_unix_test.go
@@ -10,7 +10,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/stringid"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"github.com/kr/pty"
 )
 
diff --git a/integration-cli/docker_cli_authz_unix_test.go b/integration-cli/docker_cli_authz_unix_test.go
index 0a208d7..ee469b8 100644
--- a/integration-cli/docker_cli_authz_unix_test.go
+++ b/integration-cli/docker_cli_authz_unix_test.go
@@ -20,7 +20,7 @@ import (
 	"github.com/docker/docker/pkg/authorization"
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/plugins"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 const (
diff --git a/integration-cli/docker_cli_build_test.go b/integration-cli/docker_cli_build_test.go
index a53df64..e759574 100644
--- a/integration-cli/docker_cli_build_test.go
+++ b/integration-cli/docker_cli_build_test.go
@@ -21,7 +21,7 @@ import (
 	"github.com/docker/docker/pkg/archive"
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/stringutils"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestBuildJSONEmptyRun(c *check.C) {
diff --git a/integration-cli/docker_cli_build_unix_test.go b/integration-cli/docker_cli_build_unix_test.go
index 56ab66e..179eef2 100644
--- a/integration-cli/docker_cli_build_unix_test.go
+++ b/integration-cli/docker_cli_build_unix_test.go
@@ -16,7 +16,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/go-units"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestBuildResourceConstraintsAreUsed(c *check.C) {
diff --git a/integration-cli/docker_cli_by_digest_test.go b/integration-cli/docker_cli_by_digest_test.go
index f3948d6..713d409 100644
--- a/integration-cli/docker_cli_by_digest_test.go
+++ b/integration-cli/docker_cli_by_digest_test.go
@@ -14,7 +14,7 @@ import (
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/stringutils"
 	"github.com/docker/engine-api/types"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 var (
diff --git a/integration-cli/docker_cli_commit_test.go b/integration-cli/docker_cli_commit_test.go
index 086a203..42fc2d5 100644
--- a/integration-cli/docker_cli_commit_test.go
+++ b/integration-cli/docker_cli_commit_test.go
@@ -4,7 +4,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestCommitAfterContainerIsDone(c *check.C) {
diff --git a/integration-cli/docker_cli_config_test.go b/integration-cli/docker_cli_config_test.go
index 6015231..568954b 100644
--- a/integration-cli/docker_cli_config_test.go
+++ b/integration-cli/docker_cli_config_test.go
@@ -12,7 +12,7 @@ import (
 	"github.com/docker/docker/dockerversion"
 	"github.com/docker/docker/pkg/homedir"
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestConfigHttpHeader(c *check.C) {
diff --git a/integration-cli/docker_cli_cp_from_container_test.go b/integration-cli/docker_cli_cp_from_container_test.go
index 677085a..470b498 100644
--- a/integration-cli/docker_cli_cp_from_container_test.go
+++ b/integration-cli/docker_cli_cp_from_container_test.go
@@ -5,7 +5,7 @@ import (
 	"path/filepath"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // docker cp CONTAINER:PATH LOCALPATH
diff --git a/integration-cli/docker_cli_cp_test.go b/integration-cli/docker_cli_cp_test.go
index f1ae760..d4de712 100644
--- a/integration-cli/docker_cli_cp_test.go
+++ b/integration-cli/docker_cli_cp_test.go
@@ -11,7 +11,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 const (
diff --git a/integration-cli/docker_cli_cp_to_container_test.go b/integration-cli/docker_cli_cp_to_container_test.go
index 63fbd44..1170cb6 100644
--- a/integration-cli/docker_cli_cp_to_container_test.go
+++ b/integration-cli/docker_cli_cp_to_container_test.go
@@ -4,7 +4,7 @@ import (
 	"os"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // docker cp LOCALPATH CONTAINER:PATH
diff --git a/integration-cli/docker_cli_cp_to_container_unix_test.go b/integration-cli/docker_cli_cp_to_container_unix_test.go
index 45d85ba..be4fac2 100644
--- a/integration-cli/docker_cli_cp_to_container_unix_test.go
+++ b/integration-cli/docker_cli_cp_to_container_unix_test.go
@@ -9,7 +9,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/system"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Check ownership is root, both in non-userns and userns enabled modes
diff --git a/integration-cli/docker_cli_cp_utils.go b/integration-cli/docker_cli_cp_utils.go
index 0501c5d..6da0da9 100644
--- a/integration-cli/docker_cli_cp_utils.go
+++ b/integration-cli/docker_cli_cp_utils.go
@@ -11,7 +11,7 @@ import (
 
 	"github.com/docker/docker/pkg/archive"
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 type fileType uint32
diff --git a/integration-cli/docker_cli_create_test.go b/integration-cli/docker_cli_create_test.go
index a22bb3e..5e72d19 100644
--- a/integration-cli/docker_cli_create_test.go
+++ b/integration-cli/docker_cli_create_test.go
@@ -15,7 +15,7 @@ import (
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/stringid"
 	"github.com/docker/go-connections/nat"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Make sure we can create a simple container with some args
diff --git a/integration-cli/docker_cli_daemon_experimental_test.go b/integration-cli/docker_cli_daemon_experimental_test.go
index fea887d..0e3829d 100644
--- a/integration-cli/docker_cli_daemon_experimental_test.go
+++ b/integration-cli/docker_cli_daemon_experimental_test.go
@@ -9,7 +9,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // TestDaemonRestartWithKilledRunningContainer requires live restore of running containers
diff --git a/integration-cli/docker_cli_daemon_not_experimental_test.go b/integration-cli/docker_cli_daemon_not_experimental_test.go
index 8c5634c..584e033 100644
--- a/integration-cli/docker_cli_daemon_not_experimental_test.go
+++ b/integration-cli/docker_cli_daemon_not_experimental_test.go
@@ -7,7 +7,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // os.Kill should kill daemon ungracefully, leaving behind container mounts.
diff --git a/integration-cli/docker_cli_daemon_test.go b/integration-cli/docker_cli_daemon_test.go
index 348653b..abcc5f7 100644
--- a/integration-cli/docker_cli_daemon_test.go
+++ b/integration-cli/docker_cli_daemon_test.go
@@ -25,7 +25,7 @@ import (
 	"github.com/docker/go-units"
 	"github.com/docker/libnetwork/iptables"
 	"github.com/docker/libtrust"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"github.com/kr/pty"
 )
 
diff --git a/integration-cli/docker_cli_diff_test.go b/integration-cli/docker_cli_diff_test.go
index e0d0914..797f576 100644
--- a/integration-cli/docker_cli_diff_test.go
+++ b/integration-cli/docker_cli_diff_test.go
@@ -4,7 +4,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // ensure that an added file shows up in docker diff
diff --git a/integration-cli/docker_cli_events_test.go b/integration-cli/docker_cli_events_test.go
index f17b711..1190607 100644
--- a/integration-cli/docker_cli_events_test.go
+++ b/integration-cli/docker_cli_events_test.go
@@ -14,7 +14,7 @@ import (
 
 	"github.com/docker/docker/daemon/events/testutils"
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestEventsTimestampFormats(c *check.C) {
diff --git a/integration-cli/docker_cli_events_unix_test.go b/integration-cli/docker_cli_events_unix_test.go
index c5e48f2..ca8cf1b 100644
--- a/integration-cli/docker_cli_events_unix_test.go
+++ b/integration-cli/docker_cli_events_unix_test.go
@@ -13,7 +13,7 @@ import (
 	"unicode"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"github.com/kr/pty"
 )
 
diff --git a/integration-cli/docker_cli_exec_test.go b/integration-cli/docker_cli_exec_test.go
index a8150ad..007b1a2 100644
--- a/integration-cli/docker_cli_exec_test.go
+++ b/integration-cli/docker_cli_exec_test.go
@@ -15,7 +15,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestExec(c *check.C) {
diff --git a/integration-cli/docker_cli_exec_unix_test.go b/integration-cli/docker_cli_exec_unix_test.go
index 42db409..a063afa 100644
--- a/integration-cli/docker_cli_exec_unix_test.go
+++ b/integration-cli/docker_cli_exec_unix_test.go
@@ -10,7 +10,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"github.com/kr/pty"
 )
 
diff --git a/integration-cli/docker_cli_experimental_test.go b/integration-cli/docker_cli_experimental_test.go
index 8795078..c7476cc 100644
--- a/integration-cli/docker_cli_experimental_test.go
+++ b/integration-cli/docker_cli_experimental_test.go
@@ -4,7 +4,7 @@ package main
 
 import (
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"strings"
 )
 
diff --git a/integration-cli/docker_cli_export_import_test.go b/integration-cli/docker_cli_export_import_test.go
index 069dc08..59c79e5 100644
--- a/integration-cli/docker_cli_export_import_test.go
+++ b/integration-cli/docker_cli_export_import_test.go
@@ -6,7 +6,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // export an image and try to import it into a new one
diff --git a/integration-cli/docker_cli_external_graphdriver_unix_test.go b/integration-cli/docker_cli_external_graphdriver_unix_test.go
index 33fe405..316bc54 100644
--- a/integration-cli/docker_cli_external_graphdriver_unix_test.go
+++ b/integration-cli/docker_cli_external_graphdriver_unix_test.go
@@ -17,7 +17,7 @@ import (
 	"github.com/docker/docker/daemon/graphdriver/vfs"
 	"github.com/docker/docker/pkg/archive"
 	"github.com/docker/docker/pkg/plugins"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func init() {
diff --git a/integration-cli/docker_cli_help_test.go b/integration-cli/docker_cli_help_test.go
index 93ccbeb..123115e 100644
--- a/integration-cli/docker_cli_help_test.go
+++ b/integration-cli/docker_cli_help_test.go
@@ -8,7 +8,7 @@ import (
 
 	"github.com/docker/docker/pkg/homedir"
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestHelpTextVerify(c *check.C) {
diff --git a/integration-cli/docker_cli_history_test.go b/integration-cli/docker_cli_history_test.go
index 0ee1c46..b700020 100644
--- a/integration-cli/docker_cli_history_test.go
+++ b/integration-cli/docker_cli_history_test.go
@@ -7,7 +7,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // This is a heisen-test.  Because the created timestamp of images and the behavior of
diff --git a/integration-cli/docker_cli_images_test.go b/integration-cli/docker_cli_images_test.go
index dbceddf..9720618 100644
--- a/integration-cli/docker_cli_images_test.go
+++ b/integration-cli/docker_cli_images_test.go
@@ -12,7 +12,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/stringid"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestImagesEnsureImageIsListed(c *check.C) {
diff --git a/integration-cli/docker_cli_import_test.go b/integration-cli/docker_cli_import_test.go
index 9420daf..0e33a4e 100644
--- a/integration-cli/docker_cli_import_test.go
+++ b/integration-cli/docker_cli_import_test.go
@@ -10,7 +10,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestImportDisplay(c *check.C) {
diff --git a/integration-cli/docker_cli_info_test.go b/integration-cli/docker_cli_info_test.go
index dd23694..4a545f1 100644
--- a/integration-cli/docker_cli_info_test.go
+++ b/integration-cli/docker_cli_info_test.go
@@ -7,7 +7,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/utils"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // ensure docker info succeeds
diff --git a/integration-cli/docker_cli_inspect_experimental_test.go b/integration-cli/docker_cli_inspect_experimental_test.go
index 0d9a261..2fb1649 100644
--- a/integration-cli/docker_cli_inspect_experimental_test.go
+++ b/integration-cli/docker_cli_inspect_experimental_test.go
@@ -5,7 +5,7 @@ package main
 import (
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/engine-api/types"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestInspectNamedMountPoint(c *check.C) {
diff --git a/integration-cli/docker_cli_inspect_test.go b/integration-cli/docker_cli_inspect_test.go
index 2961453..4b00859 100644
--- a/integration-cli/docker_cli_inspect_test.go
+++ b/integration-cli/docker_cli_inspect_test.go
@@ -11,7 +11,7 @@ import (
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/engine-api/types"
 	"github.com/docker/engine-api/types/container"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func checkValidGraphDriver(c *check.C, name string) {
diff --git a/integration-cli/docker_cli_kill_test.go b/integration-cli/docker_cli_kill_test.go
index 05d9a55..5091fc0 100644
--- a/integration-cli/docker_cli_kill_test.go
+++ b/integration-cli/docker_cli_kill_test.go
@@ -6,7 +6,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestKillContainer(c *check.C) {
diff --git a/integration-cli/docker_cli_links_test.go b/integration-cli/docker_cli_links_test.go
index 322b58c..5f43d2d 100644
--- a/integration-cli/docker_cli_links_test.go
+++ b/integration-cli/docker_cli_links_test.go
@@ -7,7 +7,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/runconfig"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestLinksPingUnlinkedContainers(c *check.C) {
diff --git a/integration-cli/docker_cli_links_unix_test.go b/integration-cli/docker_cli_links_unix_test.go
index 1af9279..a5f7d9e 100644
--- a/integration-cli/docker_cli_links_unix_test.go
+++ b/integration-cli/docker_cli_links_unix_test.go
@@ -7,7 +7,7 @@ import (
 	"os"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestLinksEtcHostsContentMatch(c *check.C) {
diff --git a/integration-cli/docker_cli_login_test.go b/integration-cli/docker_cli_login_test.go
index 01de75d..d5bbe9f 100644
--- a/integration-cli/docker_cli_login_test.go
+++ b/integration-cli/docker_cli_login_test.go
@@ -5,7 +5,7 @@ import (
 	"os/exec"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestLoginWithoutTTY(c *check.C) {
diff --git a/integration-cli/docker_cli_logout_test.go b/integration-cli/docker_cli_logout_test.go
index 6658da5..87c5f5c 100644
--- a/integration-cli/docker_cli_logout_test.go
+++ b/integration-cli/docker_cli_logout_test.go
@@ -7,7 +7,7 @@ import (
 	"path/filepath"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerRegistryAuthHtpasswdSuite) TestLogoutWithExternalAuth(c *check.C) {
diff --git a/integration-cli/docker_cli_logs_bench_test.go b/integration-cli/docker_cli_logs_bench_test.go
index eeb008d..c05dd71 100644
--- a/integration-cli/docker_cli_logs_bench_test.go
+++ b/integration-cli/docker_cli_logs_bench_test.go
@@ -5,7 +5,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) BenchmarkLogsCLIRotateFollow(c *check.C) {
diff --git a/integration-cli/docker_cli_logs_test.go b/integration-cli/docker_cli_logs_test.go
index e18916d..a6f08ad 100644
--- a/integration-cli/docker_cli_logs_test.go
+++ b/integration-cli/docker_cli_logs_test.go
@@ -10,7 +10,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/jsonlog"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // This used to work, it test a log of PageSize-1 (gh#4851)
diff --git a/integration-cli/docker_cli_nat_test.go b/integration-cli/docker_cli_nat_test.go
index 7f4cc2c..45496d3 100644
--- a/integration-cli/docker_cli_nat_test.go
+++ b/integration-cli/docker_cli_nat_test.go
@@ -7,7 +7,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func startServerContainer(c *check.C, msg string, port int) string {
diff --git a/integration-cli/docker_cli_netmode_test.go b/integration-cli/docker_cli_netmode_test.go
index 142f192..8266711 100644
--- a/integration-cli/docker_cli_netmode_test.go
+++ b/integration-cli/docker_cli_netmode_test.go
@@ -3,7 +3,7 @@ package main
 import (
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/runconfig"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // GH14530. Validates combinations of --net= with other options
diff --git a/integration-cli/docker_cli_network_unix_test.go b/integration-cli/docker_cli_network_unix_test.go
index 902766d..c5ef222 100644
--- a/integration-cli/docker_cli_network_unix_test.go
+++ b/integration-cli/docker_cli_network_unix_test.go
@@ -22,7 +22,7 @@ import (
 	"github.com/docker/libnetwork/ipamapi"
 	remoteipam "github.com/docker/libnetwork/ipams/remote/api"
 	"github.com/docker/libnetwork/netlabel"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"github.com/vishvananda/netlink"
 )
 
diff --git a/integration-cli/docker_cli_oom_killed_test.go b/integration-cli/docker_cli_oom_killed_test.go
index ff77f57..01c8564 100644
--- a/integration-cli/docker_cli_oom_killed_test.go
+++ b/integration-cli/docker_cli_oom_killed_test.go
@@ -4,7 +4,7 @@ package main
 
 import (
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestInspectOomKilledTrue(c *check.C) {
diff --git a/integration-cli/docker_cli_pause_test.go b/integration-cli/docker_cli_pause_test.go
index a42c2f5..f12baae 100644
--- a/integration-cli/docker_cli_pause_test.go
+++ b/integration-cli/docker_cli_pause_test.go
@@ -5,7 +5,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestPause(c *check.C) {
diff --git a/integration-cli/docker_cli_port_test.go b/integration-cli/docker_cli_port_test.go
index 80b00fe..68ca3f8 100644
--- a/integration-cli/docker_cli_port_test.go
+++ b/integration-cli/docker_cli_port_test.go
@@ -8,7 +8,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestPortList(c *check.C) {
diff --git a/integration-cli/docker_cli_proxy_test.go b/integration-cli/docker_cli_proxy_test.go
index e5699ca..9a131ed 100644
--- a/integration-cli/docker_cli_proxy_test.go
+++ b/integration-cli/docker_cli_proxy_test.go
@@ -6,7 +6,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestCliProxyDisableProxyUnixSock(c *check.C) {
diff --git a/integration-cli/docker_cli_ps_test.go b/integration-cli/docker_cli_ps_test.go
index 72650ff..c2e7b78 100644
--- a/integration-cli/docker_cli_ps_test.go
+++ b/integration-cli/docker_cli_ps_test.go
@@ -13,7 +13,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/stringid"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestPsListContainersBase(c *check.C) {
diff --git a/integration-cli/docker_cli_pull_local_test.go b/integration-cli/docker_cli_pull_local_test.go
index 1f85832..b96d9d5 100644
--- a/integration-cli/docker_cli_pull_local_test.go
+++ b/integration-cli/docker_cli_pull_local_test.go
@@ -16,7 +16,7 @@ import (
 	"github.com/docker/distribution/manifest/manifestlist"
 	"github.com/docker/distribution/manifest/schema2"
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // testPullImageWithAliases pulls a specific image tag and verifies that any aliases (i.e., other
diff --git a/integration-cli/docker_cli_pull_test.go b/integration-cli/docker_cli_pull_test.go
index ac211d2..81a8bbb 100644
--- a/integration-cli/docker_cli_pull_test.go
+++ b/integration-cli/docker_cli_pull_test.go
@@ -9,7 +9,7 @@ import (
 
 	"github.com/docker/distribution/digest"
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // TestPullFromCentralRegistry pulls an image from the central registry and verifies that the client
diff --git a/integration-cli/docker_cli_pull_trusted_test.go b/integration-cli/docker_cli_pull_trusted_test.go
index 6bc38e6..25e628f 100644
--- a/integration-cli/docker_cli_pull_trusted_test.go
+++ b/integration-cli/docker_cli_pull_trusted_test.go
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerTrustSuite) TestTrustedPull(c *check.C) {
diff --git a/integration-cli/docker_cli_push_test.go b/integration-cli/docker_cli_push_test.go
index 6b3d823..8bf9cea 100644
--- a/integration-cli/docker_cli_push_test.go
+++ b/integration-cli/docker_cli_push_test.go
@@ -15,7 +15,7 @@ import (
 	"github.com/docker/distribution/digest"
 	"github.com/docker/docker/cliconfig"
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Pushing an image to a private registry.
diff --git a/integration-cli/docker_cli_registry_user_agent_test.go b/integration-cli/docker_cli_registry_user_agent_test.go
index 67a950c..5ab294c 100644
--- a/integration-cli/docker_cli_registry_user_agent_test.go
+++ b/integration-cli/docker_cli_registry_user_agent_test.go
@@ -5,7 +5,7 @@ import (
 	"net/http"
 	"regexp"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // unescapeBackslashSemicolonParens unescapes \;()
diff --git a/integration-cli/docker_cli_rename_test.go b/integration-cli/docker_cli_rename_test.go
index cbb60f8..1c3dfa7 100644
--- a/integration-cli/docker_cli_rename_test.go
+++ b/integration-cli/docker_cli_rename_test.go
@@ -5,7 +5,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/stringid"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestRenameStoppedContainer(c *check.C) {
diff --git a/integration-cli/docker_cli_restart_test.go b/integration-cli/docker_cli_restart_test.go
index 31c8920..7bca267 100644
--- a/integration-cli/docker_cli_restart_test.go
+++ b/integration-cli/docker_cli_restart_test.go
@@ -7,7 +7,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestRestartStoppedContainer(c *check.C) {
diff --git a/integration-cli/docker_cli_rm_test.go b/integration-cli/docker_cli_rm_test.go
index 0186c56..73c5658 100644
--- a/integration-cli/docker_cli_rm_test.go
+++ b/integration-cli/docker_cli_rm_test.go
@@ -5,7 +5,7 @@ import (
 	"os"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestRmContainerWithRemovedVolume(c *check.C) {
diff --git a/integration-cli/docker_cli_rmi_test.go b/integration-cli/docker_cli_rmi_test.go
index 697be32..c251d51 100644
--- a/integration-cli/docker_cli_rmi_test.go
+++ b/integration-cli/docker_cli_rmi_test.go
@@ -8,7 +8,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/stringid"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestRmiWithContainerFails(c *check.C) {
diff --git a/integration-cli/docker_cli_run_test.go b/integration-cli/docker_cli_run_test.go
index 6e45fca..eee0d94 100644
--- a/integration-cli/docker_cli_run_test.go
+++ b/integration-cli/docker_cli_run_test.go
@@ -25,7 +25,7 @@ import (
 	"github.com/docker/go-connections/nat"
 	"github.com/docker/libnetwork/netutils"
 	"github.com/docker/libnetwork/resolvconf"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	libcontainerUser "github.com/opencontainers/runc/libcontainer/user"
 )
 
diff --git a/integration-cli/docker_cli_run_unix_test.go b/integration-cli/docker_cli_run_unix_test.go
index fe7cc69..9ddac61 100644
--- a/integration-cli/docker_cli_run_unix_test.go
+++ b/integration-cli/docker_cli_run_unix_test.go
@@ -21,7 +21,7 @@ import (
 	"github.com/docker/docker/pkg/mount"
 	"github.com/docker/docker/pkg/parsers"
 	"github.com/docker/docker/pkg/sysinfo"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"github.com/kr/pty"
 )
 
diff --git a/integration-cli/docker_cli_save_load_test.go b/integration-cli/docker_cli_save_load_test.go
index b21c987..2dd3106 100644
--- a/integration-cli/docker_cli_save_load_test.go
+++ b/integration-cli/docker_cli_save_load_test.go
@@ -15,7 +15,7 @@ import (
 
 	"github.com/docker/distribution/digest"
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // save a repo using gz compression and try to load it using stdout
diff --git a/integration-cli/docker_cli_save_load_unix_test.go b/integration-cli/docker_cli_save_load_unix_test.go
index d9dd95f..5cdba85 100644
--- a/integration-cli/docker_cli_save_load_unix_test.go
+++ b/integration-cli/docker_cli_save_load_unix_test.go
@@ -10,7 +10,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"github.com/kr/pty"
 )
 
diff --git a/integration-cli/docker_cli_search_test.go b/integration-cli/docker_cli_search_test.go
index dfab810..f7a988b 100644
--- a/integration-cli/docker_cli_search_test.go
+++ b/integration-cli/docker_cli_search_test.go
@@ -4,7 +4,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // search for repos named  "registry" on the central registry
diff --git a/integration-cli/docker_cli_sni_test.go b/integration-cli/docker_cli_sni_test.go
index fb896d5..cba8d97 100644
--- a/integration-cli/docker_cli_sni_test.go
+++ b/integration-cli/docker_cli_sni_test.go
@@ -10,7 +10,7 @@ import (
 	"os/exec"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestClientSetsTLSServerName(c *check.C) {
diff --git a/integration-cli/docker_cli_start_test.go b/integration-cli/docker_cli_start_test.go
index 4334219..aedfd4d 100644
--- a/integration-cli/docker_cli_start_test.go
+++ b/integration-cli/docker_cli_start_test.go
@@ -6,7 +6,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Regression test for https://github.com/docker/docker/issues/7843
diff --git a/integration-cli/docker_cli_start_volume_driver_unix_test.go b/integration-cli/docker_cli_start_volume_driver_unix_test.go
index cc5d0b1..c511cf0 100644
--- a/integration-cli/docker_cli_start_volume_driver_unix_test.go
+++ b/integration-cli/docker_cli_start_volume_driver_unix_test.go
@@ -17,7 +17,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/engine-api/types"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func init() {
diff --git a/integration-cli/docker_cli_stats_test.go b/integration-cli/docker_cli_stats_test.go
index e3c7a3e..97861e2 100644
--- a/integration-cli/docker_cli_stats_test.go
+++ b/integration-cli/docker_cli_stats_test.go
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestStatsNoStream(c *check.C) {
diff --git a/integration-cli/docker_cli_tag_test.go b/integration-cli/docker_cli_tag_test.go
index 1e60152..e290999 100644
--- a/integration-cli/docker_cli_tag_test.go
+++ b/integration-cli/docker_cli_tag_test.go
@@ -7,7 +7,7 @@ import (
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/stringid"
 	"github.com/docker/docker/pkg/stringutils"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // tagging a named image in a new unprefixed repo should work
diff --git a/integration-cli/docker_cli_top_test.go b/integration-cli/docker_cli_top_test.go
index e0865b9..3cf9ace 100644
--- a/integration-cli/docker_cli_top_test.go
+++ b/integration-cli/docker_cli_top_test.go
@@ -4,7 +4,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestTopMultipleArgs(c *check.C) {
diff --git a/integration-cli/docker_cli_update_test.go b/integration-cli/docker_cli_update_test.go
index 188030f..5a0178b 100644
--- a/integration-cli/docker_cli_update_test.go
+++ b/integration-cli/docker_cli_update_test.go
@@ -5,7 +5,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestUpdateRestartPolicy(c *check.C) {
diff --git a/integration-cli/docker_cli_update_unix_test.go b/integration-cli/docker_cli_update_unix_test.go
index c40ad3e..e4267b1 100644
--- a/integration-cli/docker_cli_update_unix_test.go
+++ b/integration-cli/docker_cli_update_unix_test.go
@@ -9,7 +9,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/engine-api/types"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestUpdateRunningContainer(c *check.C) {
diff --git a/integration-cli/docker_cli_userns_test.go b/integration-cli/docker_cli_userns_test.go
index f8b3f77..d14d5d3 100644
--- a/integration-cli/docker_cli_userns_test.go
+++ b/integration-cli/docker_cli_userns_test.go
@@ -13,7 +13,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/system"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // user namespaces test: run daemon with remapped root setting
diff --git a/integration-cli/docker_cli_v2_only_test.go b/integration-cli/docker_cli_v2_only_test.go
index 889936a..72438e9 100644
--- a/integration-cli/docker_cli_v2_only_test.go
+++ b/integration-cli/docker_cli_v2_only_test.go
@@ -6,7 +6,7 @@ import (
 	"net/http"
 	"os"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func makefile(contents string) (string, func(), error) {
diff --git a/integration-cli/docker_cli_version_test.go b/integration-cli/docker_cli_version_test.go
index 7672beb..16d02c9 100644
--- a/integration-cli/docker_cli_version_test.go
+++ b/integration-cli/docker_cli_version_test.go
@@ -4,7 +4,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // ensure docker version works
diff --git a/integration-cli/docker_cli_volume_test.go b/integration-cli/docker_cli_volume_test.go
index a10316f..4bb0c3b 100644
--- a/integration-cli/docker_cli_volume_test.go
+++ b/integration-cli/docker_cli_volume_test.go
@@ -5,7 +5,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestVolumeCliCreate(c *check.C) {
diff --git a/integration-cli/docker_cli_wait_test.go b/integration-cli/docker_cli_wait_test.go
index 2993397..4eb7f3a 100644
--- a/integration-cli/docker_cli_wait_test.go
+++ b/integration-cli/docker_cli_wait_test.go
@@ -7,7 +7,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // non-blocking wait with 0 exit code
diff --git a/integration-cli/docker_experimental_network_test.go b/integration-cli/docker_experimental_network_test.go
index f33dbd1..3437702 100644
--- a/integration-cli/docker_experimental_network_test.go
+++ b/integration-cli/docker_experimental_network_test.go
@@ -9,7 +9,7 @@ import (
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/parsers/kernel"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 var (
diff --git a/integration-cli/docker_hub_pull_suite_test.go b/integration-cli/docker_hub_pull_suite_test.go
index 6aa9346..9533cc3 100644
--- a/integration-cli/docker_hub_pull_suite_test.go
+++ b/integration-cli/docker_hub_pull_suite_test.go
@@ -6,7 +6,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func init() {
diff --git a/integration-cli/docker_utils.go b/integration-cli/docker_utils.go
index 3198287..e9e2452 100644
--- a/integration-cli/docker_utils.go
+++ b/integration-cli/docker_utils.go
@@ -30,7 +30,7 @@ import (
 	"github.com/docker/engine-api/types"
 	"github.com/docker/go-connections/tlsconfig"
 	"github.com/docker/go-units"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func init() {
diff --git a/integration-cli/events_utils.go b/integration-cli/events_utils.go
index cdd3106..cc534db 100644
--- a/integration-cli/events_utils.go
+++ b/integration-cli/events_utils.go
@@ -12,7 +12,7 @@ import (
 	"github.com/Sirupsen/logrus"
 	"github.com/docker/docker/daemon/events/testutils"
 	"github.com/docker/docker/pkg/integration/checker"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // eventMatcher is a function that tries to match an event input.
diff --git a/integration-cli/registry.go b/integration-cli/registry.go
index fa3fb87..e039771 100644
--- a/integration-cli/registry.go
+++ b/integration-cli/registry.go
@@ -9,7 +9,7 @@ import (
 	"path/filepath"
 
 	"github.com/docker/distribution/digest"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 const (
diff --git a/integration-cli/registry_mock.go b/integration-cli/registry_mock.go
index 300bf46..cf0c77a 100644
--- a/integration-cli/registry_mock.go
+++ b/integration-cli/registry_mock.go
@@ -7,7 +7,7 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 type handlerFunc func(w http.ResponseWriter, r *http.Request)
diff --git a/integration-cli/requirements.go b/integration-cli/requirements.go
index e948c0a..74d77c8 100644
--- a/integration-cli/requirements.go
+++ b/integration-cli/requirements.go
@@ -9,7 +9,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 type testCondition func() bool
diff --git a/integration-cli/trust_server.go b/integration-cli/trust_server.go
index 77314c3..dbdde32 100644
--- a/integration-cli/trust_server.go
+++ b/integration-cli/trust_server.go
@@ -14,7 +14,7 @@ import (
 	"github.com/docker/docker/cliconfig"
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/tlsconfig"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 var notaryBinary = "notary"
diff --git a/pkg/discovery/discovery_test.go b/pkg/discovery/discovery_test.go
index 6084f3e..b1a479c 100644
--- a/pkg/discovery/discovery_test.go
+++ b/pkg/discovery/discovery_test.go
@@ -3,7 +3,7 @@ package discovery
 import (
 	"testing"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Hook up gocheck into the "go test" runner.
diff --git a/pkg/discovery/file/file_test.go b/pkg/discovery/file/file_test.go
index 667f00b..03b6cb1 100644
--- a/pkg/discovery/file/file_test.go
+++ b/pkg/discovery/file/file_test.go
@@ -7,7 +7,7 @@ import (
 
 	"github.com/docker/docker/pkg/discovery"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Hook up gocheck into the "go test" runner.
diff --git a/pkg/discovery/generator_test.go b/pkg/discovery/generator_test.go
index 6281c46..ba73a19 100644
--- a/pkg/discovery/generator_test.go
+++ b/pkg/discovery/generator_test.go
@@ -1,7 +1,7 @@
 package discovery
 
 import (
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DiscoverySuite) TestGeneratorNotGenerate(c *check.C) {
diff --git a/pkg/discovery/kv/kv_test.go b/pkg/discovery/kv/kv_test.go
index 4fe5239..656e7c8 100644
--- a/pkg/discovery/kv/kv_test.go
+++ b/pkg/discovery/kv/kv_test.go
@@ -12,7 +12,7 @@ import (
 	"github.com/docker/libkv"
 	"github.com/docker/libkv/store"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Hook up gocheck into the "go test" runner.
diff --git a/pkg/discovery/memory/memory_test.go b/pkg/discovery/memory/memory_test.go
index c2da0a0..70c6786 100644
--- a/pkg/discovery/memory/memory_test.go
+++ b/pkg/discovery/memory/memory_test.go
@@ -4,7 +4,7 @@ import (
 	"testing"
 
 	"github.com/docker/docker/pkg/discovery"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Hook up gocheck into the "go test" runner.
diff --git a/pkg/discovery/nodes/nodes_test.go b/pkg/discovery/nodes/nodes_test.go
index e26568c..a6b8fea 100644
--- a/pkg/discovery/nodes/nodes_test.go
+++ b/pkg/discovery/nodes/nodes_test.go
@@ -5,7 +5,7 @@ import (
 
 	"github.com/docker/docker/pkg/discovery"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Hook up gocheck into the "go test" runner.
diff --git a/pkg/integration/checker/checker.go b/pkg/integration/checker/checker.go
index a531499..a33808f 100644
--- a/pkg/integration/checker/checker.go
+++ b/pkg/integration/checker/checker.go
@@ -2,7 +2,7 @@
 package checker
 
 import (
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"github.com/vdemeester/shakers"
 )
 
diff --git a/pkg/integration/dockerCmd_utils.go b/pkg/integration/dockerCmd_utils.go
index fab3e06..54ea8cc 100644
--- a/pkg/integration/dockerCmd_utils.go
+++ b/pkg/integration/dockerCmd_utils.go
@@ -6,7 +6,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // We use the elongated quote mechanism for quoting error returns as
diff --git a/pkg/integration/dockerCmd_utils_test.go b/pkg/integration/dockerCmd_utils_test.go
index 3dd5d11..a5900db 100644
--- a/pkg/integration/dockerCmd_utils_test.go
+++ b/pkg/integration/dockerCmd_utils_test.go
@@ -10,7 +10,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 const dockerBinary = "docker"
